src/components/FilterDropdown.vue

```diff
@@ -306,7 +306,17 @@ export default {
       }
 
       if (state.shouldSort) {
-        temp = temp.sort((a, b) => (isSelected(a) > isSelected(b) ? -1 : 1))
+        temp = temp.sort((a, b) => {
+          if (a.text > b.text) return -1
+          else if (a.text < b.text) return 1
+          else return 0
+        })
```

Comment:
Wouldn't `localeCompare` be better here?
```suggestion
        temp = temp.sort((a, b) => a.localeCompare(b))
```

I am actually not 100% but that's my gut feeling.

Reply:
this one seems to produce the same results as the original implementation:

```
temp = temp.sort((a, b) => b.text?.localeCompare(a.text))
```